.circleci/config.yml

```diff
@@ -1,13 +1,45 @@
 version: 2.1
 
 orbs:
-  haskell: haskell-works/haskell-build@1.6.3
-  github: haskell-works/github-release@1.2.1
-  hackage: haskell-works/hackage@1.0.0
+  haskell: haskell-works/haskell-build@1.6.4
+  github: haskell-works/github-release@1.3.0
+  hackage: haskell-works/hackage@1.1.0
+
+executors:
+  darwin:
+    macos:
+      xcode: 10.0
 
 workflows:
   multiple-ghc-build:
     jobs:
+      - haskell/build:
+          name: Darwin
+          executor: darwin
+          run-tests: false
+          fail-incoherent-builds: false
+          write-result-workspace: true
+          after-checkout:
+            - restore_cache:
+                keys:
+                  - homebrew-cache
+            - run:
+                name: Install GHC
+                command: brew install jq md5sha1sum ghc cabal-install
+
+            - save_cache:
+                key: homebrew-cache
+                # I cache Homebrew itself, not the stuff that is installed with it
+                # because brew update takes up to 4 minutes for some reason.
+                paths: [/usr/local/Homebrew]
+
+            - run:
+                name: Install cabal-plan
+                command: |
+                  curl -L https://github.com/haskell-works/cabal-plan/releases/download/v0.5.0.0/cabal-plan_x86_osx.tar.gz -o cabal-plan_x86_osx.tar.gz
```

Comment:
Is there a way to go via `/releases/download/latest/` or anything like that?

Reply:
Not directly, but possible. 
In my understanding we can do an API call to get the latest release info, and then use its `tag` property to construct this URL.